native_libs/src/2d_array.c

```diff
@@ -104,6 +104,37 @@ char **drop_row(char **mat, size_t rows, size_t cols, size_t stride_r, size_t st
     return new_mat;
 }
 
+char **join(char **fst_mat, char **snd_mat, size_t fst_rows, size_t fst_cols, size_t snd_rows, size_t snd_cols){
+    size_t cols = (((fst_cols) > (snd_cols)) ? (fst_cols) : (snd_cols));
```

Comment:
Too many parentheses here, `(fst_cols > snd_cols) ? fst_cols : snd_cols;` is ok.